python/cudf/cudf/pylibcudf_tests/test_lists.py

```diff
@@ -44,3 +47,72 @@ def test_concatenate_list_elements(test_data, dropna, expected):
     expect = pa.array(expected)
 
     assert_column_eq(expect, res)
+
+
+def test_contains_scalar(test_data):
+    list_column = test_data[0][0]
+    arr = pa.array(list_column)
+
+    plc_column = plc.interop.from_arrow(arr)
+    scalar = pa.scalar(1)
+    plc_scalar = plc.interop.from_arrow(scalar)
```

Comment:
We generally try and fixturize any GPU side test data, I believe for performance reasons